components/aws/sagemaker/tests/integration_tests/Dockerfile

```diff
@@ -38,6 +38,7 @@ ENV S3_DATA_BUCKET="kfp-test-data"
 ENV MINIO_LOCAL_PORT=9000
 ENV KFP_NAMESPACE="kubeflow"
 
-COPY ./sagemaker/ .
+RUN mkdir kfp_repo
```

Comment:
lets name it pipelines -> same as repository?

Reply:
done